js/main.js

```diff
@@ -18,19 +18,20 @@ $(document).ready( function() {
   });
 });
 
-// This is a conditional (Ternary) Operator. It assigns a value based on some other value
-// It is much shorter to write than if-else.
-// How it works:
-// 		localStorage allows you to store values in the client side. Its like cookies
-//		but they don't get send to the server
-//		1. If 'visited' is already defined:
-//			a. get the 'visited' and convert it to a number (+) since
-//			   localStorage stores it as a string
-//			b. Add +1 to it
-//		2. If 'visited' is not defined (null), set it to 1, since you are on the website
-//	   	   right now.
-//		3. Assign the value you chose back to the 'visited' in localStorage
-localStorage['visited'] = localStorage['visited'] ? +localStorage['visited']+1 : 1;
+/*  This is a conditional (Ternary) Operator. It assigns a value based on some other value
```

Comment:
out of interest do we need a function explaining what a Ternary is?

Lot's of the comments here are just explaining what the code does. and should probably be removed since they are dead air